Gemfile

```diff
@@ -16,10 +16,10 @@ gem 'logstasher'
 gem 'mechanize'
 gem 'momentjs-rails', '~> 2.5.0'
 gem 'nokogiri', '~> 1.10'
-gem 'paper_trail', '~> 10.3.1'
+gem 'paper_trail', '~> 11.0.0'
 gem 'pg', '~> 1.2'
 gem 'puma', '~> 4.3'
-gem 'rails', '~> 5.2'
+gem 'rails', '~> 6.0'
```

Comment:
@digitalali-moj  not sure whether it is worthy spending time to have a look at the notes from the following link
https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-2-to-rails-6-0